app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -129,7 +129,7 @@ const PageHome: React.FunctionComponent<PageHomeProps> = () => {
               role="img"
               aria-label="download icon"
             />
-            <span> Reporting</span>
```

Comment:
Nit: should we also change the testId `reporting-anchor` to `report-anchor`? (line 122)

Reply:
that's a good point, thanks @eliserichards will make a change